replaceSchemaFile.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+)
+
+const packageVersion = "version210"
+const jsonSchemaVersion = "JsonSchema210"
+const filePath = "./pkg/devfile/parser/data/v2/2.1.0/devfileJsonSchema210.go"
+
+func ReplaceSchemaFile() {
+	originalSchema := os.Args[2]
+	schemaURL := os.Args[3]
+
+	newSchema := strings.ReplaceAll(originalSchema, "`", "'")
+	fmt.Printf("Writing to file: %s\n", filePath)
+	fileContent := fmt.Sprintf("package %s\n\n// %s\nconst %s = `%s\n`\n", packageVersion, schemaURL, jsonSchemaVersion, newSchema)
+
+	if err := ioutil.WriteFile(filePath, []byte(fileContent), 0755); err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
```

Comment:
Is this only to meant to update current schema? How about 2.2.0? I think we also need to update the const in versions.go for 2.2.0 and may not be possible 🤔  and does that mean new version is always manual?

Reply:
For new versions, we need to set up once and update the const value in this file

Reply:
ok, then we can store paths, schema info and share logic with this https://github.com/devfile/library/pull/64#discussion_r585023117?

Reply:
store all info in `./updateApi.sh`, and allow to pass an extra argument to update schema for selected devfile version
(now only has case 2.0.0, 2.1.0 will be default case)